### ISSUE TITLE
[BUG] - Non-existent - Attempted to add item 222698 to category Cooking

### ISSUE THREAD

**Issue by dsoundmn** [AvyAddons/BetterBags_WarWithin_Crafting#2]
### Description
```lua
Message: Interface/AddOns/BetterBags/data/categories.lua:140: Attempted to add item 222698 to category Cooking, but the item does not exist.
Time: Mon Aug 26 19:24:30 2024
Count: 1
Stack: Interface/AddOns/BetterBags/data/categories.lua:140: Attempted to add item 222698 to category Cooking, but the item does not exist.
[string "@Interface/AddOns/BetterBags/data/categories.lua"]:140: in function `AddItemToCategory'
[string "@Interface/AddOns/BetterBags_TWW_Crafting/main.lua"]:14: in main chunk

Locals: self = <table> {
 modules = <table> {
 }
 defaultModuleLibraries = <table> {
 }
 categoryCount = 0
 name = "BetterBags_Categories"
 ephemeralCategories = <table> {
 }
 orderedModules = <table> {
 }
 categoryFunctions = <table> {
 }
 baseName = "BetterBags"
 ephemeralCategoryByItemID = <table> {
 }
 enabledState = false
 defaultModuleState = true
 moduleName = "Categories"
 itemsWithNoCategory = <table> {
 }
}
id = 222698
category = "Cooking"
database = <table> {
 modules = <table> {
 }
 defaultModuleState = true
 data = <table> {
 }
 enabledState = true
 baseName = "BetterBags"
 defaultModuleLibraries = <table> {
 }
 name = "BetterBags_Database"
 orderedModules = <table> {
 }
 moduleName = "Database"
}
```

### Addon Version

v1.0.0

### World of Warcraft Flavor

Retail (The War Within)

### Tested with only BetterBags_TWW_Crafting

- [X] Yes

### Reproduction Steps

occurs on log-on.

**Comment by dsoundmn** [comment]
thanks for the amazing app - as an aside - no issues occur following the failure - please let me know if i should try anything!

**Comment by lucasvienna** [comment]
Thanks for the report, I've fixed it in v1.0.1!

**Comment by dsoundmn** [comment]
awesome! thank you! - tested and it's all set.
